### PR TITLE
Fix configs

### PR DESCRIPTION
**Pull request by melroy89**
- Fix bare metal/VM configs
- Fix Docker configs

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -41,7 +41,11 @@ HCAPTCHA_SECRET=
 
 # Redis
 REDIS_PASSWORD=!ChangeThisRedisPass!
-REDIS_DNS=redis://${REDIS_PASSWORD}@redis
+REDIS_DNS=redis://${REDIS_PASSWORD}@127.0.0.1:6379
+# Or when using Redis Socket:
+#REDIS_DNS=REDIS_DNS=redis://${REDIS_PASSWORD}/var/run/redis/redis-server.sock
+# Or socket without password:
+#REDIS_DNS=REDIS_DNS=redis:///var/run/redis/redis-server.sock
```
Comment (e-five256) [inline .env.example:48]: was it intentional the env vars were duplicated on both these likes? `REDIS_DNS=REDIS_DNS=`

Comment (melroy89) [reply .env.example:48]: No it was not thanks for spotting.

**Review by nobodyatroot — APPROVED** [review]
Approved after matrix conversation to update mercure port in docker config.

**Review by nobodyatroot — CHANGES_REQUESTED** [review]
See comment above

**Review by nobodyatroot — DISMISSED** [review]

**Review by nobodyatroot — DISMISSED** [review]
Verified docker changes in local testing environment.

**Review by nobodyatroot — APPROVED** [review]

**Pull request merged**